src/common.rs

```diff
@@ -139,19 +137,19 @@ impl std::fmt::Display for AccountTransferAllowance {
 }
 
 impl AccountTransferAllowance {
-    pub fn liquid_storage_stake(&self) -> NearBalance {
-        NearBalance::from_yoctonear(
+    pub fn liquid_storage_stake(&self) -> near_token::NearToken {
+        near_token::NearToken::from_yoctonear(
```

Comment:
No need to reconstruct NearToken from yoctonearto perform math operations on NearToken, there are dedicated methods implemented:

https://docs.rs/near-token/latest/near_token/struct.NearToken.html#method.saturating_sub

Reply:
Done